dataland-frontend/cypress.config.ts

```diff
@@ -28,6 +28,7 @@ export default defineConfig({
             ? process.env.DATA_REQUEST_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTES
             : "2"
     },
+    scrollBehavior: "center",
```

Comment:
What was the reason to make elements being scrolled to the center of the viewport per default?
Simply convenvience (to see stuff in the center when running cypress tests locally during dev-work?).

Reply:
There were some tests broken. The reason was that in App.vue there was `scroll-behavior: smooth` newly set. It seems ot have fiddled with the cypress settings and scrolling in tests was broken. This property scrolls to the current element (retrieved via `cy.get` et al.)

Reply:
Take out as soon as CI is green, and then check if it stays green

Reply:
Removed it and all tests went well.